### PR TITLE
Allow set info FileDispositionInformation

### PR DESCRIPTION
**Pull request by sopak**
Add ability to set DeleteOnClose on already opened file.

### CODE REVIEW

File: src/main/java/com/hierynomus/msfscc/fileinformation/FileInformationFactory.java [hunk 1]
```diff
@@ -107,6 +107,25 @@ public void write(FileBasicInformation info, Buffer outputBuffer) {
...
+            }
+
+            @Override
+            public FileDispositionInformation read(Buffer inputBuffer) throws Buffer.BufferException {
+                throw new UnsupportedOperationException();
+            }
+
+            @Override
+            public void write(FileDispositionInformation info, Buffer outputBuffer) {
+                outputBuffer.putRawBytes(FileInformationFactory.getFileDispositionInfo(true));
```
Comment (hierynomus) [inline src/main/java/com/hierynomus/msfscc/fileinformation/FileInformationFactory.java]: Shouldn't the boolean option be encoded in the `FileDispositionInformation` class?

Comment (sopak) [inline src/main/java/com/hierynomus/msfscc/fileinformation/FileInformationFactory.java]: Do you mean set it also to false?
As if one operation will delete the file and another will discard it on the same opened fileId?

Comment (hierynomus) [inline src/main/java/com/hierynomus/msfscc/fileinformation/FileInformationFactory.java]: No, using `false` you can also unmark it for deletion. See: http://download.microsoft.com/download/4/3/8/43889780-8d45-4b2e-9d3a-c696a890309f/File%20System%20Behavior%20Overview.pdf

**Comment by sopak** [comment]
I did something wrong here. I will create a new pull request.

**Pull request closed without merging**